integration/server.js

```diff
@@ -3,10 +3,14 @@ const Fastify = require('../fastify')
 const fastify = Fastify()
 
 fastify.listen({
-  host: '::',
+  host: 'localhost',
```

Comment:
why this change?

Reply:
We have reverted the change. However, Our current branch only runs the onListen hooks when the host is assigned to 'localhost'. We are currently working on updating the functionality to be more universal (i.e. '::'). Is this something we should have done before this PR is merged or possibly after in another PR?